en/starter/generator/command_line_tools.md

```diff
@@ -0,0 +1,154 @@
+
+The command-line tools provide an [Application generator](#application-generator) to create a new Swift Server Generator application and a number of sub-generators to define models and properties within the application, as described in the following table:
+
+*Table 1. Available commands*
+
+| Command                           | Command type                                   | Description                                                                                                                                                                                                                                                            |
+|---------------------------------  |------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`yo swiftserver`**              | [Application generator](#application-generator) | Create a new Swift Server Generator application.                                                                                                                                                                                                                       |
+| **`yo create swiftserver:model`** | [Model generator](#model-generator)             | Add a new model to a Swift Server Generator application.                                                                                                                                                                                                               |
+| **`yo swiftserver:property`**     | [Property generator](#property-generator)       | Add a new property to an existing model.                                                                                                                                                                                                                               |
+| **`yo swiftserver:refresh`**      | Refresh generator                              | Generate OpenAPI (Swagger 2.0) and product definition YAML files, by convention named *`application`*`.yaml` and *`application`*`-product.yaml`. If the product file already exists it is not regenerated. The OpenAPI (Swagger 2.0) definition is always regenerated. |
+
+---
+
+## Application generator
+
+**Synopsis**
+
+Creates a new Swift Server Generator application.
+
+    yo swiftserver [options] [name]
+
+**Options**
+
+    -h, --help
+
+Print the generator’s options and usage.
+
+    -n, --name
+
+Provide the application name.

```

Comment:
There is no `-n` or `--name` option. The name can be passed as an optional argument.